docs/standard/garbage-collection/fundamentals.md

```diff
@@ -150,57 +150,45 @@ Before a garbage collection starts, all managed threads are suspended except for
-
-- By default, there is an affinity between the server GC heap and a CPU. That is, in server GC, the garbage collector assumes that a process owns the machine on which it is running, so it uses all available CPUs for garbage collection. It creates a dedicated heap, a GC thread, and, if background garbage collection is enabled, a background GC thread for each processor. This can result in poor performance, particularly on systems with multiple running instances of a server application. To not affinitize server GC threads with CPUs, use the [\<GCNoAffinitize>](../../framework/configure-apps/file-schema/runtime/gcnoaffinitize-element.md) setting in .NET Framework applications.
-
-- You can limit the number of heaps created by the garbage collector by using the [\<GCHeapCount](../../framework/configure-apps/file-schema/runtime/gcheapcount-element.md) setting in .NET Framework applications. If GC thread/processor affinity is disabled, this setting limits the number of GC heaps. If GC thread/processor affinity is enabled, this setting limits the number of GC heaps to the processors 0 to one-less-than its specified value.
+![Server Garbage Collection Threads](./media/gc-server.png)
 
-- If processor affinity is enabled, you can use the [\<GCHeapAffinitizeMask>](../../framework/configure-apps/file-schema/runtime/gcheapaffinitizemask-element.md) configuration element in .NET Framework applications to control the specific processors for which a GC heap and threads are created. Supply a decimal value. The value is a mask that defines the processors that are available to the process.
-
-### Comparing workstation and server garbage collection
+### Compare workstation and server garbage collection
 
 The following are threading and performance considerations for workstation garbage collection:
 
 - The collection occurs on the user thread that triggered the garbage collection and remains at the same priority. Because user threads typically run at normal priority, the garbage collector (which runs on a normal priority thread) must compete with other threads for CPU time.
 
   Threads that are running native code are not suspended.
 
-- Workstation garbage collection is always used on a computer that has only one processor, regardless of the [\<gcServer>](../../../docs/framework/configure-apps/file-schema/runtime/gcserver-element.md) setting. If you specify server garbage collection, the CLR uses workstation garbage collection with concurrency disabled.
+- Workstation garbage collection is always used on a computer that has only one processor, regardless of the [gcServer](../../../docs/framework/configure-apps/file-schema/runtime/gcserver-element.md) setting in .NET Framework apps or the `System.GC.Server` setting in .NET Core apps. If you specify server garbage collection on such a computer, the CLR uses workstation garbage collection with concurrency disabled.
```

Comment:
>If you specify server garbage collection on such a computer, the CLR uses workstation garbage collection with concurrency disabled [](start = 284, length = 130)

this is not true anymore (has never been true on core). we used to do this because there was no concurrent for Server GC.